e2e-tests/insurance/cypress/e2e/journeys/policy/pre-credit-period/pre-credit-period.spec.js

```diff
@@ -210,6 +210,25 @@ context(`Insurance - Policy - Pre-credit period page - ${story}`, () => {
           fieldSelector(CREDIT_PERIOD_WITH_BUYER).textarea().should('have.value', expectedValue);
         });
       });
+
+      // TODO: EMS-2621 - move this test into "check/change" answers.
```

Comment:
Note - we don't have "check/change" answers for this field yet. Next PR will be adding this functionality. So for now the test coverage is in here.